python_modules/dagster/dagster/_grpc/proxy_server.py

```diff
@@ -121,22 +131,22 @@ def _reload_location(self):
             self._logger.exception("Failure while loading code")
 
         if self._client:
-            self._heartbeat_shutdown_event = threading.Event()
-            self._heartbeat_thread = threading.Thread(
+            self._client_heartbeat_shutdown_event = threading.Event()
```

Comment:
rename these properties because now it's confusing that there's a client and server heartbeat thread